x-pack/plugins/ux/common/elasticsearch_fieldnames.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
there is already a copy in the obs plugin, so this will be the 3rd one - any chance we can consolidate? maybe as a separate package?

Reply:
definitely, a follow up. i think we will move this to observability plugin. i am not sold on moving it to a package.